src/Navigation/Resumedownload/index.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import "./index.css";
 function Resume(props, ref) {
   return (
-    <h2 ref={ref} className="resume-download">
-      Resume
-    </h2>
+    <div ref={ref} className="resume-download">
+      <div className="resume-download_logo"></div>
+      <a
+        className="resume-download_heading"
+        href={"https://registry.jsonresume.org/ShilpiMaurya"}
```

Comment:
This is not a downloadable link. Please look here
https://www.w3schools.com/howto/howto_html_download_link.asp

Also, make resume.json downloadable.